weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallsPage/ManageColumnsButton.tsx

```diff
@@ -0,0 +1,192 @@
+/**
+ * Button/popup for grid column visibility state.
+ */
+
+import {Popover} from '@mui/material';
+import {GridColDef, GridColumnVisibilityModel} from '@mui/x-data-grid-pro';
+import {Switch} from '@wandb/weave/components';
+import classNames from 'classnames';
+import React, {useCallback, useRef, useState} from 'react';
+
+import {Button} from '../../../../../Button';
+import {DraggableGrow} from '../../../../../DraggablePopups';
+import {TextField} from '../../../../../Form/TextField';
+import {Tailwind} from '../../../../../Tailwind';
+import {ColumnInfo} from '../../types';
+import {TraceCallSchema} from '../wfReactInterface/traceServerClient';
+
+type ManageColumnsButtonProps = {
+  columnInfo: ColumnInfo;
+  columnVisibilityModel: GridColumnVisibilityModel;
+  setColumnVisibilityModel: (model: GridColumnVisibilityModel) => void;
+};
+
+export const ManageColumnsButton = ({
+  columnInfo,
+  columnVisibilityModel,
+  setColumnVisibilityModel,
+}: ManageColumnsButtonProps) => {
+  const [search, setSearch] = useState('');
+  const ref = useRef<HTMLDivElement>(null);
+  const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null);
+  const onClick = (event: React.MouseEvent<HTMLElement>) => {
+    setAnchorEl(anchorEl ? null : ref.current);
+    setSearch('');
+  };
+
+  const open = Boolean(anchorEl);
+  const id = open ? 'simple-popper' : undefined;
+
+  const toggleColumnShow = useCallback(
+    (key: string) => {
+      const on = columnVisibilityModel[key] ?? true;
+      const newModel = {
+        ...columnVisibilityModel,
+        [key]: !on,
+      };
+      setColumnVisibilityModel(newModel);
+    },
+    [columnVisibilityModel, setColumnVisibilityModel]
+  );
+  const onHideAll = () => {
+    const newModel = columnInfo.cols.reduce((acc, col) => {
+      if (col.hideable ?? true) {
+        acc[col.field] = false;
+      }
+      return acc;
+    }, {} as GridColumnVisibilityModel);
+    setColumnVisibilityModel(newModel);
+  };
+  const onShowAll = () => {
+    const newModel = columnInfo.cols.reduce((acc, col) => {
+      // If a column is not hideable, we also don't need to explicitly show it.
+      if (col.hideable ?? true) {
+        acc[col.field] = true;
+      }
+      return acc;
+    }, {} as GridColumnVisibilityModel);
+    setColumnVisibilityModel(newModel);
+  };
+
+  return (
+    <>
+      <span ref={ref}>
+        <Button
+          variant="ghost"
+          icon="column"
+          tooltip="Manage columns"
+          onClick={onClick}
+        />
+      </span>
+      <Popover
```

Comment:
this looks so good. One comment is that when dragging around the window my mouse turns into a horizontal scroll icon, i might expect a grabber. like: 

```
cursor: grabbing;
```

or something like that.

Reply:
like its currently: 
![image](https://github.com/wandb/weave/assets/19414170/407a61ae-dca1-45ed-9ba3-7d7ba2a53596)

Reply:
Nice suggestion. I don't know why react-draggable is using `ew-resize` for this. I'll override it.

Reply:
After this PR goes in I can update our other popups to have the same grabbing cursor behavior.